types/koa-logger/index.d.ts

```diff
@@ -1,13 +1,14 @@
-// Type definitions for koa-logger v2.0
+// Type definitions for koa-logger 3.1
```

Comment:
I'm confused about the version upgrade. The changes below look good from a cleanliness perspective, but there's no change in the actual typings. Why not leave it at the lower version, since it doesn't look like anything's changed?

Reply:
Good point, in fact nothing has been changed between that versions in the end-user perspective.

But while other types packages are in most cases representing the package version why should this one be different?

I mean while installing `koa-logger@^3.1.0` an an user I would like to also have types in version `@types/koa-logger@^3.1.0` to not confuse users who don't know about that similarities. Does it make sense?

Reply:
Sounds good to me